### ISSUE TITLE
Hard dependency on Phoenix 1.3.2

### ISSUE THREAD

**Issue by tmjoen** [ueberauth/guardian_phoenix#4]
```
Failed to use "phoenix" (versions 1.4.0 to 1.4.9) because
  guardian_phoenix (version 2.0.0) requires 1.3.2
  phoenix_live_reload (version 1.2.1) requires ~> 1.4
  mix.exs specifies ~> 1.4
```

Looks like an oversight in `mix.exs`:
https://github.com/ueberauth/guardian_phoenix/blob/1d2664367ab617884c7964a97d623546a807aac3/mix.exs#L54

**Comment by yordis** [comment]
Definitely shouldn't be `1.3.2`, but I am not sure about `1.4` Phoenix since they had some breaking changes.

Would you mind to take the lead testing it with Phoenix 1.4?

**Comment by gil-air-may** [comment]
Any news on this issue?

**Comment by doomspork** [comment]
@tmjoen or @gil-air-may have you had a chance to test this with 1.4?

@yordis we could release a patch version with the dep moved to `1.3`. Thoughts?

**Comment by yordis** [comment]
Yeah, we could at least use `~> 1.3` instead

**Comment by tmjoen** [comment]
Seems to work fine with 1.4 in my app. I don't use `Permissions` though, so YMMV.

**Comment by yordis** [comment]
@tmjoen are you getting any warnings? I think we need to fix some warnings.

**Comment by yordis** [comment]
I released `2.0.1`, using `~> 1.3` instead.

Let me know if that works or if I have to add `~> 1.4` as well
